package/src/components/SelectableItem/v1/SelectableItem.js

```diff
@@ -147,29 +210,61 @@ class SelectableItem extends Component {
     return this.state.value !== this.props.value;
   }
 
+  renderIcon() {
+    const { components } = this.props;
+    const { iconAmericanExpress, iconVisa, iconDiscover, iconMastercard } = components;
+    let icon;
+    if (this.props.item.icon === "iconAmericanExpress") {
+      icon = iconAmericanExpress;
+    } else if (this.props.item.icon === "iconVisa") {
+      icon = iconVisa;
+    } else if (this.props.item.icon === "iconDiscover") {
+      icon = iconDiscover;
+    } else if (this.props.item.icon === "iconMastercard") {
+      icon = iconMastercard;
+    }
+
+    return (
+      <span className="icon">
+        {icon}
+      </span>
+    );
+  }
+
   render() {
-    const { className, label, name, detail } = this.props;
-    const { id, value } = this.state;
+    const {
+      name,
+      item: {
+        _id,
+        className,
+        label,
+        detail,
+        icon
+      }
+    } = this.props;
+    const { value } = this.state;
 
     return (
       <StyledItem className={className} >
         <input
+          id={_id}
           checked={value === true}
-          id={id}
+          key={_id}
           onChange={this.onChange}
           type="radio"
           name={name}
```

Comment:
I believe the `name` attribute is not necessary, this form's data will not be posted by a `form` element.